core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -185,6 +186,31 @@ public void testRequestMessage() throws Exception {
         assertNotNull(requestMessage.getRawPayload());
     }
 
+    @Test
+    public void testRequestMessageWithTags() throws Exception {
```

Comment:
purpose of tests - will ensure the request is tagged with the dynamic and static tags. name of test does say not enough